test/project_types/script/layers/application/push_script_test.rb

```diff
@@ -57,6 +57,9 @@
       .returns(task_runner)
     ShopifyCLI::Environment.stubs(:interactive?).returns(true)
 
+    Script::Layers::Application::ProjectDependencies
+      .stubs(:install).with(ctx: @context, task_runner: task_runner)
```

Comment:
I believe this is better suited to be an expectation, WDYT? Installing dependencies is a side affect that we should probably ensure is happening at the right time

Reply:
Yeah I agree. I moved it up to a stub to remove some of the duplication and avoid having to specify it everytime but I'll revert.